spread.yaml

```diff
@@ -458,6 +458,8 @@ debug-each: |
                 echo "# device cgroup $cgroup_dev"
                 cat "/sys/fs/cgroup/devices/$cgroup_dev/devices.list" || true
             fi
+        else
+            ls -l /sys/fs/bpf/snap || true
```

Comment:
I suggest having an `echo` command before this line

Reply:
added